src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/helpdoc.txt

```diff
@@ -0,0 +1,630 @@
+    --query                       : JMESPath query string. See http://jmespath.org/ for more
+                                    information and examples.
+    --verbose                     : Increase logging verbosity. Use --debug for full debug logs.
+
+(env) vishrut@mac azure-cli (devtestlabs-docs) $ az lab vm create -h
+
+Command
+    az lab vm create: Command to create vm of in Azure DevTest Lab.
+
+Arguments
+    --lab-name          [Required]: Name of the Lab.
+    --name -n           [Required]: Name of the virtual machine.
+    --resource-group -g [Required]: Name of the lab resource group.
+    --allow-claim                 : Indicates whether another user can take ownership of the virtual
+                                    machine.
+    --artifacts                   : JSON encoded array of artifacts to be applied. Use @{file} to
+                                    load from a file.
+    --disk-type                   : Storage type to use for virtual machine (i.e. Standard,
+                                    Premium).
+    --expiration-date             : The expiration date in UTC(YYYY-mm-dd) for VM. Ex. 2017-03-25.
+    --formula                     : Name of the formula. Use az lab formula list for available
+                                    formulas.
+    --image                       : The name of the operating system image (Gallery Image Name and
+                                    Custom Image Name/ID). Use az lab gallery-image list for
+                                    available Gallery Images or Use az lab custom-image list for
+                                    available Custom Images.
+    --image-type                  : Type of the image. Allowed values are gallery, custom.
+    --location -l                 : Location where to create VM. Defaults to the location of the
+                                    lab.
+    --notes                       : Notes for the virtual machine.
+    --size                        : The VM size to be created. See https://azure.microsoft.com/en-
+                                    us/pricing/details/virtual-machines/ for size info.
+    --tags                        : Space separated tags in 'key[=value]' format. Use "" to clear
+                                    existing tags.
+
+Authentication Arguments
+    --admin-password              : Password for the VM.
+    --admin-username              : Username for the VM.  Default: vishrut.
+    --authentication-type         : Type of authentication to use with the VM. Allowed values are
+                                    password, ssh.  Default: password.
+    --generate-ssh-keys           : Generate SSH public and private key files if missing.
+    --ssh-key                     : The SSH public key or public key file path. Use --generate-ssh-
+                                    keys to regen ssh keys.
+
+Network Arguments
+    --disallow-public-ip-address  : To allow public ip address set to true or false. If omitted,
```

Comment:
Yes i am working on the PR and should be out soon for review. Here is an updated docs for the command
```
(env) vishrut@mac azure-cli (ipconfig) $ az lab vm create -h

Command
    az lab vm create: Command to create VM in a DevTest Lab.

Arguments
    --lab-name          [Required]: Name of the Lab.
    --name -n           [Required]: Name of the virtual machine.
    --resource-group -g [Required]: Name of lab's resource group.
    --allow-claim                 : Flag indicating whether the VM will be created as a claimable VM
                                    in the lab.
    --artifacts                   : JSON encoded array of artifacts to be applied. Use @{file} to
                                    load from a file.
    --disk-type                   : Storage type to use for virtual machine (i.e. Standard,
                                    Premium).
    --expiration-date             : The expiration date in UTC(YYYY-mm-dd) for VM. Ex. 2017-03-25.
    --formula                     : Name of the formula. Use az lab formula list for available
                                    formulas. Use --artifacts to pass-in parameters required by the
                                    artifacts of the formula.
    --image                       : The name of the operating system image (Gallery Image Name and
                                    Custom Image Name/ID). Use az lab gallery-image list for
                                    available Gallery Images or Use az lab custom-image list for
                                    available Custom Images.
    --image-type                  : Type of the image. Allowed values are gallery, custom.
    --notes                       : Notes for the virtual machine.
    --size                        : The VM size to be created. See https://azure.microsoft.com/en-
                                    us/pricing/details/virtual-machines/ for size info. Required
                                    when creating VM from gallery or custom image.
    --tags                        : Space separated tags in 'key[=value]' format. Use "" to clear
                                    existing tags.

Authentication Arguments
    --admin-password              : Password for the VM.
    --admin-username              : Username for the VM.  Default: vishrut.
    --authentication-type         : Type of authentication to use with the VM. Allowed values are
                                    password, ssh.  Default: password.
    --generate-ssh-keys           : Generate SSH public and private key files if missing.
    --ssh-key                     : The SSH public key or public key file path. Use --generate-ssh-
                                    keys to regen ssh keys.

Network Arguments
    --ip-configuration            : Type of ip configuration to use with the VM. Allowed values are
                                    shared, public, private. If omitted, will be selected based on
                                    selected vnet.
    --subnet                      : Name of the subnet to reference an existing one in lab. If
                                    omitted, lab's existing one subnet will be selected
                                    automatically.
    --vnet-name                   : Name of the virtual network to reference an existing one in lab.
                                    If omitted, lab's existing one VNet and subnet will be selected
                                    automatically.

Global Arguments
    --debug                       : Increase logging verbosity to show all debug logs.
    --help -h                     : Show this help message and exit.
    --output -o                   : Output format.  Allowed values: json, jsonc, table, tsv.
                                    Default: json.
    --query                       : JMESPath query string. See http://jmespath.org/ for more
                                    information and examples.
    --verbose                     : Increase logging verbosity. Use --debug for full debug logs.

Examples
    Create a Virtual Machine in the lab from gallery image.
        az lab vm create --lab-name MyLab -g MyRG --name MyVM --image "Ubuntu Server 16.04 LTS"
        --image-type gallery --size Standard_DS1_v2


    Create a Virtual Machine in the lab from gallery image with ssh authentication.
        az lab vm create --lab-name MyLab -g MyRG --name MyVM --image "Ubuntu Server 16.04 LTS"
        --image-type gallery --size Standard_DS1_v2 --authentication-type ssh


    Create a claimable Virtual Machine in the lab from gallery image with password authentication.
        az lab vm create --lab-name MyLab -g MyRG --name MyVM --image "Ubuntu Server 16.04 LTS"
        --image-type gallery --size Standard_DS1_v2 --allow-claim


    Create a windows Virtual Machine in the lab from gallery image with password authentication.
        az lab vm create --lab-name MyLab -g MyRG --name MyVM --image "Windows Server 2008 R2 SP1"
        --image-type gallery --size Standard_DS1_v2


    Create a Virtual Machine in the lab from custom image.
        az lab vm create --lab-name MyLab -g MyRG --name MyVM --image "jenkins_custom" --image-type
        custom --size Standard_DS1_v2


    Create a Virtual Machine in the lab with public ip configuration.
        az lab vm create --lab-name MyLab -g MyRG --name MyVM --image "Ubuntu Server 16.04 LTS"
        --image-type gallery --size Standard_DS1_v2 --ip-configuration public


    Create a Virtual Machine from a formula.
        az lab vm create --lab-name MyLab -g MyRG --name MyVM --formula MyFormula
```